sentences/nb/cover_HassTurnOn.yaml

```diff
@@ -2,20 +2,20 @@ language: nb
 intents:
   HassTurnOn:
     data:
+      - sentences:
+          - "<åpne> <navn> [<i_på> <område>]"
+        requires_context:
+          domain: cover
+        response: cover
       - sentences:
           - "<åpne> <garasje>"
         slots:
           domain: cover
           device_class: garage
         response: cover_device_class
       - sentences:
-          - "<åpne> [<vindu>|<gardin>|<rullegardin>|<skodde>] <i_på> <område>"
-          - "<åpne> <område>[s][ ][<vindu>|<gardin>|<rullegardin>|<skodde>]"
+          - "<åpne> [the] {cover_classes:device_class} <i_på> <område>"
```

Comment:
`the`? I'm guessing a copy-paste error?

Reply:
Yes, I have added a commit to remove the [the]